src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -107,7 +107,13 @@ public CosmosDBTriggerAttributeBindingProvider(INameResolver nameResolver, Cosmo
                 throw new InvalidOperationException(string.Format("Cannot create Collection Information for {0} in database {1} with lease {2} in database {3} : {4}", attribute.CollectionName, attribute.DatabaseName, attribute.LeaseCollectionName, attribute.LeaseDatabaseName, ex.Message), ex);
             }
 
-            return new CosmosDBTriggerBinding(parameter, documentCollectionLocation, leaseCollectionLocation, leaseHostOptions);
+            // -1 is allowed and indicates "infinite retries", but other negative values are not valid
+            if (attribute.RetryCount < -1)
+            {
+                throw new InvalidOperationException(string.Format("RetryCount value {0} is not valid, value must be either positive, or -1 for 'infinite retries'", attribute.RetryCount));
```

Comment:
minor nit: we try to use string interpolation for new strings we add -- and `nameof()` for property names so you can change this to: `$"{nameof(RetryCount} value {attribute.RetryCount} is not valid. Value must be either positive, or -1 for infinite retries."`

Reply:
The other strings in the class (for the branch/version of code I am looking at) are all string.Format'ed rather than using $ and I don't see any other nameof() calls so I assume that's in another branch or something?  Anyway, I've made the requested change and pushed.